.github/workflows/release.yaml

```
@@ -1,13 +1,15 @@
-name: deploy docs
+name: Release
 
 on:
   push:
-    branches:
-      - main
+    tags: ["[0-9].[0-9]+.[0-9]+.*"]
```

This tag didn't work properly on my machine, so I just made it the following again.
```suggestion
    tags: ["*"]
```

---

I updated it differently: 

`tags: ["[0-9].[0-9]+.[0-9]+*"]`